Signal/src/ViewControllers/MessagesViewController.m

```diff
@@ -1856,15 +1866,38 @@ - (void)collectionView:(JSQMessagesCollectionView *)collectionView
                             }
                         }
                     }
+                } else if ([messageItem.media isKindOfClass:[AttachmentPointerAdapter class]]) {
```

Comment:
Probably not now, but smells ripe for some good ole [replace conditional with polymorphism](https://refactoring.com/catalog/replaceConditionalWithPolymorphism.html)